spark/src/main/scala/org/apache/spark/sql/delta/commands/DeleteWithDeletionVectorsHelper.scala

```diff
@@ -181,8 +181,12 @@ object DeleteWithDeletionVectorsHelper extends DeltaCommand {
     val (dvAddFiles, dvRemoveFiles) = dvUpdates.unzip
     val dvAddFilesWithStats = getActionsWithStats(spark, dvAddFiles, snapshot)
 
-    // TODO: gather more metrics
-    val metricMap = Map("numDeletedRows" -> numDeletedRows, "numRemovedFiles" -> numRemovedFiles)
+    val numDeletionVectorsAdded: Long = dvAddFiles.count(_.deletionVector != null)
```

Comment:
the `AddFile` may already have DVs which gets replaced with a new DV? Should we consider the removed existing DVs as part of the `numDeletionVectorsRemoved`?

Reply:
I would be in favour of tracking this separately as `numDeletionVectorsUpdated`. That makes it clearer what's going on.

Reply:
@larsk-db I disagree with the use of ```numDeletionVectorsUpdated``` to track the deletion of a DV.

When a delete operation is performed, a new DV is created to replace it.
It's similar to a delete operation on a table without DV. when someone uses the delete command and we have to delete some rows from a file. We write a new file without those rows, and in delta history, we show it as ```numRemovedFiles -> 1``` and ```numAddedFiles -> 1```

The use of ```numDeletionVectorsUpdated``` would make it seem as though the DV is being updated, when in reality it is being replaced.

I'm open to feedback on my argument. If you think it's not sound, will go with ```numDeletionVectorsUpdated```

Reply:
I can see where you are coming from.
The way I look at it: From the perspective of the `AddFile` entry the DV belongs to, it's basically an update to replace a DV.
The most important thing to me are the semantics to keep the 3 counters separate. Beyond that...a name is just a name ;)

Reply:
ok will go with variable named ```numVecsSentToShadowRealm```